samples/xss-bot/healthcheck/Makefile

```diff
@@ -0,0 +1,26 @@
+# Copyright 2020 Google LLC
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     https://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+SHELL := bash
+
+# IMPORTANT
+# - This will be the entrypoint when building the docker image.
+.gen/docker-image: .gen/new-image-id
+	NEW_ID=$$(cat .gen/new-image-id | cut -d ':' -f 2); if [[ "$${NEW_ID}" != $$(cat $@ 2>/dev/null) ]]; then echo "$${NEW_ID}" > $@; fi
```

Comment:
People actually need to type all this for every task? :-/

Reply:
I'm not sure if I understand, when would they type this?

Reply:
when making a new task, I mean. it looks a bit too magical.

can we instead just use the command below as the only required rule?

Reply:
when making a new task, they write kctf-chal-create $chal_name. That copies the Makefile from the challenge skeleton in here so there's no typing involved.
Do you mean it's an ugly command to have in here and if we could move it to the global makefile?

Reply:
yea, so having something like:
```
.gen/new-image-id:
docker build . > $@
```
is clear what is doing and why, but the bash comparison/cutting and such is not super trivial to understand

Reply:
yeah, good idea.

Reply:
done (see #60)